auth/src/main/java/com/redhat/lightblue/rest/auth/ldap/LdapFindUserByUidCommand.java

```diff
@@ -78,7 +84,7 @@ This methods is executed for all types of failure such as run() failure, timeout
      */
     @Override
     protected SearchResult getFallback() {
-        LOGGER.info("Error during the execution of the command. Falling back to the cache");
+        LOGGER.error("Error during the execution of the command. Falling back to the cache");
```

Comment:
Should we log this at error level or something lower, maybe warn?  Because we have a fallback, seems it could be a warning and if the fallback fails we'd have an error.